commands/create.go

```diff
@@ -110,7 +110,18 @@ func create(command *Command, args *Args) {
 	gh := github.NewClient(project.Host)
 
 	if gh.IsRepositoryExist(project) {
-		ui.Errorln("Existing repository detected. Updating git remote")
+
+		repo, err := gh.Repository(project)
+		if err != nil {
+			utils.Check(err)
+		}
+
+		if !repo.Private && flagCreatePrivate {
+			err = fmt.Errorf("Existing public repository detected. git remote not updated")
```

Comment:
I would like to see a clearer error message, like:

```go
fmt.Errorf("Repository `%s' already exists and is public", repo.FullName)
```

Reply:
I wasn't quite sure about the messaging, I'll go with your suggestion